src/main/java/de/muenchen/isi/configuration/StateMachineConfiguration.java

```diff
@@ -0,0 +1,128 @@
+                .event(StatusAbfrageEvents.SPEICHERN_DER_VARIANTEN)
+                .and()
+                .withExternal()
+                .source(StatusAbfrage.IN_ERFASSUNG).target(StatusAbfrage.ABBRUCH)
+                .event(StatusAbfrageEvents.ABBRECHEN)
+                .and()
+                .withExternal()
+                .source(StatusAbfrage.IN_ERFASSUNG).target(StatusAbfrage.ANGELEGT)
+                .event(StatusAbfrageEvents.ANGABEN_ANPASSEN)
+                .and()
+
+                // Alle Events vom Status IN_BEARBEITUNG_PLAN
+                .withExternal()
+                .source(StatusAbfrage.IN_BEARBEITUNG_PLAN).target(StatusAbfrage.ERLEDIGT)
+                .event(StatusAbfrageEvents.KEINE_BEARBEITUNG_NOETIG)
+                .and()
+                .withExternal()
+                .source(StatusAbfrage.IN_BEARBEITUNG_PLAN).target(StatusAbfrage.IN_BEARBEITUNG_FACHREFERATE)
+                .event(StatusAbfrageEvents.VERSCHICKEN_DER_STELLUNGNAHME)
+                .and()
+                .withExternal()
+                .source(StatusAbfrage.IN_BEARBEITUNG_PLAN).target(StatusAbfrage.ABBRUCH)
+                .event(StatusAbfrageEvents.ABBRECHEN)
+                .and()
+                .withExternal()
+                .source(StatusAbfrage.IN_BEARBEITUNG_PLAN).target(StatusAbfrage.IN_ERFASSUNG)
+                .event(StatusAbfrageEvents.WEITERE_ABFRAVARIANTEN_ANLEGEN)
+                .and()
+
+                //Alle Events vom Status IN_BEARBEITUNG_FACHREFERATE
+                .withExternal()
+                .source(StatusAbfrage.IN_BEARBEITUNG_FACHREFERATE).target(StatusAbfrage.BEDARFSMELDUNG_ERFOLGT)
+                .event(StatusAbfrageEvents.BEDARFSMELDUNG_ERFOLGTE)
+                .and()
+                .withExternal()
+                .source(StatusAbfrage.IN_BEARBEITUNG_FACHREFERATE).target(StatusAbfrage.ABBRUCH)
+                .event(StatusAbfrageEvents.ABBRECHEN)
+                .and()
+                .withExternal()
+                .source(StatusAbfrage.IN_BEARBEITUNG_FACHREFERATE).target(StatusAbfrage.IN_ERFASSUNG)
+                .event(StatusAbfrageEvents.WEITERE_ABFRAVARIANTEN_ANLEGEN)
+                .and()
+
+                // Alle Events vom Status BEDARFSMELDUNG_ERFOLGT
+                .withExternal()
+                .source(StatusAbfrage.BEDARFSMELDUNG_ERFOLGT).target(StatusAbfrage.ERLEDIGT)
+                .event(StatusAbfrageEvents.SPEICHERN_VON_SOZIALINFRASTRUKTUR_VERSORGUNG)
+                .and()
+                .withExternal()
+                .source(StatusAbfrage.BEDARFSMELDUNG_ERFOLGT).target(StatusAbfrage.ABBRUCH)
+                .event(StatusAbfrageEvents.ABBRECHEN)
+                .and()
+                .withExternal()
+                .source(StatusAbfrage.BEDARFSMELDUNG_ERFOLGT).target(StatusAbfrage.IN_ERFASSUNG)
+                .event(StatusAbfrageEvents.WEITERE_ABFRAVARIANTEN_ANLEGEN)
+                .and()
+                // Alle Events von ERLDEDIGT
```

Comment:
Leerzeile davor einfügen.
```suggestion

                // Alle Events von ERLDEDIGT
```